libpod/options.go

```diff
@@ -322,8 +324,8 @@ func WithSELinuxLabels(processLabel, mountLabel string) CtrCreateOption {
 	}
 }
 
-// WithUser sets the user identity field in configutation
-// Valid uses [user | user:group | uid | uid:gid | user:gid | uid:group ]
+// WithUser sets the user identity field in configutation.
```

Comment:
While we're touching this line, we may want to fix "configutation" -> "configuration".  Or maybe we just want to drop "in configutation" or similar?  I think:

> WithUser sets the user identity.  Valid formats include: user, user:group, uid, uid:gid, user:gid, and uid:group.

or similar would be a reasonable godoc for this function.

This is also getting fairly far into the weeds, so I'm fine punting on any or all of the godoc cleanup in this PR.